prelexer.cpp

```diff
@@ -130,7 +130,7 @@ namespace Sass {
 
 
     const char* interpolant(const char* src) {
-      return delimited_by<hash_lbrace, rbrace, false>(src);
```

Comment:
IMO almost all uses of `delimited_by` are erroneous. We should probably aim to wholesale replace this with something smarter. 

https://github.com/sass/libsass/issues/615 is good example where `delimited_by` fails spectacularly.